packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -290,10 +292,7 @@ export const BlocksDropdown = ({ disabled }) => {
    * @param {string} optionKey - key of the heading selected
    */
   const selectOption = (optionKey) => {
-    if (optionKey === 'image') {
-      // Image node created using select or existing selection node needs to be deleted before adding new image nodes
-      Transforms.removeNodes(editor);
-    } else if (['list-ordered', 'list-unordered'].includes(optionKey)) {
```

Comment:
If you remove this condition it will go back to have a prob of image validation. That means it will just toggle the block with type: image which is wrong.

Reply:
I moved the condition where we insert directly the image, maybe we can discuss the issue together tomorrow, thanks